### PR TITLE
editorial cleanups

### PR DESCRIPTION
**Pull request by bradh**
Mostly spelling fixes.

A few `rel` links were updated to match http://docs.opengeospatial.org/DRAFTS/19-072.html#link-relation-conventions

### CODE REVIEW

File: core/standard/requirements/tileset/REQ_tile-template.adoc [hunk 1]
```diff
@@ -6,5 +6,5 @@
 ^|B |These links SHALL provide a URL template with the fragment `/tiles` followed by the value of the tileMatrixSet identifier and a URI template that makes use of the following URL template variables {tileMatrix}, {tileRow}, {tileCol}. Once the variables are substituted by their respective valid values, a URL to a tile is obtained.
 ^|C |There SHALL be a link to a tile URI template for each file format that the server supports (the format is indicated in the 'type' attribute of the link)
 ^|D |A property `templated` SHALL be part of the link properties to indicate that the link needs to be processed to substitute the templated variables with valid values before being used as a URL to a tile.
-^|E |The results of this operation SHALL return a single tile that represents information coming from geospatial data resources.
+^|E |The results of this operation SHALL return a single tile that represents information provided from geospatial data resources.
```
Comment (jerstlouis) [inline core/standard/requirements/tileset/REQ_tile-template.adoc:9]: _originating_ ?

File: openapi/swaggerHubUnresolved/ogc-api-tiles-opf-xmp-vt-more-1-collection.yaml [hunk 2]
```diff
@@ -92,25 +92,25 @@ paths:
...
                   - 'http://www.opengis.net/spec/ogcapi-features-1/1.1/req/oas30'
                   - 'http://www.opengis.net/spec/ogcapi-features-1/1.1/req/html'
                   - 'http://www.opengis.net/spec/ogcapi-features-1/1.1/req/geojson'
                   # OGC API Tiles core consists on the capability to serve a tiles from a collection 
-                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/req/core'
-                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/req/tms'
-                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/req/collections'
+                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/conf/core'
+                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/conf/tms'
+                  - 'http://www.opengis.net/spec/ogcapi-tiles-1/1.0/conf/collections'
```
Comment (jerstlouis) [inline openapi/swaggerHubUnresolved/ogc-api-tiles-opf-xmp-vt-more-1-collection.yaml:113]: This list of conformance classes is out of date...
The _Tiles_ conformance classes now are:
- tileset (we are talking about bringing back "core", by splitting everything but the "tileset metadata/description" out of tileset)
- tilesets-list (depends on: tileset)
- dataset-tilesets (depends on: tileset, tilesets-list)
- geodata-tilesets (depends on: tileset, tilesets-list)
- geodata-selection (we are discussing this `collections=` query parameter likely belongs in Common Part 2)

File: core/standard/clause_7_tile_tileSet.adoc [hunk 3]
```diff
@@ -142,7 +140,7 @@ A tile resource is a geospatial resource of a fragment of a more bigger geospati
 As described before a tile path is obtained by extracting a tile URL templated from one of the links with `rel`: `item` in tileset description document and substituting the templated variables of with valid values.
 
 ==== Operation
-This operation allows retrieving a single tile that represents information coming from geospatial data resources.
+This operation allows retrieving a single tile that represents information provided from geospatial data resources.
```
[the remark this answers is not in the document]
Comment (bradh) [reply core/standard/clause_7_tile_tileSet.adoc:143]: Maybe. Suggests something like "original source" though.

File: core/standard/clause_7_tile_tileSet.adoc [hunk 4]
```diff
@@ -131,8 +131,6 @@ The following table explains the meaning of the URI template variables.
 |TileCol |column index of tile matrix	|A non-negative integer between 0 and the MatrixWidth – 1. If there is a TileMatrixSetLimits the value is limited between MinTileCol and MaxTileCol
 !===
 
-NOTE: The use of "templated" is inspired by the JSON Hypertext Application Language (HAL), https://tools.ietf.org/html/draft-kelly-json-hal-08
-
```
[the remark this answers is not in the document]
Comment (bradh) [reply core/standard/clause_7_tile_tileSet.adoc:135]: This is a duplicate of a statement that is already included (just above, line 120).

File: core/standard/requirements/tileset/REQ_tile-template.adoc (hunk 1 above)
Comment (bradh) [reply core/standard/requirements/tileset/REQ_tile-template.adoc:9]: As long as its the same.

File: openapi/swaggerHubUnresolved/ogc-api-tiles-opf-xmp-vt-more-1-collection.yaml (hunk 2 above)
Comment (bradh) [reply openapi/swaggerHubUnresolved/ogc-api-tiles-opf-xmp-vt-more-1-collection.yaml:113]: Follow-on fix?

File: core/standard/clause_7_tile_tileSet.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (jerstlouis) [reply core/standard/clause_7_tile_tileSet.adoc:135]: Ah! Right. Thanks, was not obvious from the git diff.

File: core/standard/clause_7_tile_tileSet.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (jerstlouis) [reply core/standard/clause_7_tile_tileSet.adoc:143]: how about "sourced" from then?

File: core/standard/requirements/tileset/REQ_tile-template.adoc (hunk 1 above)
Comment (bradh) [reply core/standard/requirements/tileset/REQ_tile-template.adoc]: OK.

File: openapi/swaggerHubUnresolved/ogc-api-tiles-opf-xmp-vt-more-1-collection.yaml (hunk 2 above)
Comment (bradh) [reply openapi/swaggerHubUnresolved/ogc-api-tiles-opf-xmp-vt-more-1-collection.yaml]: Backed out the `ogcapi-tiles-1` changes

File: core/standard/clause_7_tile_tileSet.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (bradh) [reply core/standard/clause_7_tile_tileSet.adoc:143]: OK

**Comment by jerstlouis** [comment]
Thanks @bradh !

**Pull request merged**